### ISSUE TITLE
DAO formation and tooling

### ISSUE THREAD

**Issue by randomishwalk** [cryptoShire/project-management#12]
Looking for folks with experience forming DAOs, configuring tooling around voting/governance/proposals (i.e. Aragon or using MolochDAO), and familiarity with tokenomics + want to help us bootstrap an exciting new project in the asset management space :)

**Comment by muddlebee** [comment]
I like Aragon's dispute resolution protocol https://aragon.org/aragon-court

Code:  https://github.com/aragon/aragon-court

**Comment by randomishwalk** [comment]
Yes agree. Aragon seems to have great tooling. Gnosis is another one (the multisig safe) that I'm looking into
